openseeds/ols-9/index.md

```diff
@@ -0,0 +1,365 @@
+---
+layout: page
+toc: true
+title: Welcome to Open Seeds Cohort 9
+image: /images/syllabus.jpg
+photos:
+  name: Niklas Morberg
+  license: CC BY-NC 2.0
+  url: https://flic.kr/p/5BXB6s
+redirect_from:
+  - /OLS-9
+  - /ols-9
```

Comment:
```suggestion
```

Reply:
Not needed as it was not in the previous structure

Reply:
True. 🤔💯 

I did imagine a situation where someone would attempt to update an old url to find the new cohort "ols-9."